src/analysis/multiSpecies/microbiomeModelingToolbox/mgPipe/README.md

```diff
@@ -16,33 +16,36 @@ function.  [PART 3] Simulations under different diet regimes. Set of standard
 analysis to apply to the personalized models. PCA of computed MNPCs of
 individuals as for example.
 
-**WARNING :**Please take into consideration only the files listed in this
+**WARNING:** Please take into consideration only the files listed in this
 document. Everything present in the folder but not listed and explained in this
 document is to be considered not relevant or obsolete.
 
 ## Requirements
 
-MgPipe requires Matrix Laboratory as well as the Cobra Toolbox to be installed
-with ILOG CPLEX as solver.  All toolboxes/solvers/files should be in the MATLAB
-path.  MgPipe was created (and tested) for AGORA 1.0 please first download
-AGORA version 1.0 from https://vmh.uni.lu/#downloadview and place the mat files
-into a dedicated folder.
+MgPipe requires `Matrix Laboratory`, the `Parallel Computing Toolbox`, as well
+as, the Toolbox to be installed with the `ILOG CPLEX` solver. Pleas refer to
+the [installation instructions](https://opencobra.github.io/cobratoolbox/stable/installation.html).
```

Comment:
Just as a question:
Apart from reproducibility or numeric stability:
Why do you need CPLEX? Shouldn't any other solver do the same job?

Reply:
Hi Thomas,
The reason for asking for CPLEX is to be able to use fastFVA function which, at the moment, is not compatible with other solvers. As matter of fact, it is possible to run the simulations also with the fluxVariability function and this works with every solver, but it is much slower....

Reply:
Then I would suggest to indicate that, and only say that it is suggested to use CPLEX due to fastFVA not being available for other solvers and thus slower. Even though from some quick tests, on recon2, they are not really far apart speedwise (which might be due to a lot of reactions in recon 2 having their ub as max and lb as min flux, which speeds up fluxVariability a lot).